src/protobuf-net.Core/Helpers.cs

```diff
@@ -32,39 +32,53 @@ internal static MethodInfo GetInstanceMethod(Type declaringType, string name, Ty
         internal static bool IsSubclassOf(Type type, Type baseClass)
             => type.IsSubclassOf(baseClass);
 
-        public static ProtoTypeCode GetTypeCode(Type type)
+        public static ProtoTypeCode GetTypeCode(Type type, bool resolveNullableValueType = true)
```

Comment:
I'm concerned with the default of this being `true`, since that means the *default* is to change the behaviour, making it harder for me to review the contexts in which this changes; IMO the default here should be `false`, then we can *opt in* at the places where we **intend** to change the behaviour.

At the moment, the PR only tells me "here's two places (L1236 and L2380) that **behave exactly like they used to**" - this is the *uninteresting* thing to me; what I'm interested in is the places that **now behave differently**.

Can we reverse this default? or make it non-default?

Reply:
Sure I will change it, makes perfect sense.